### DISCUSSION TITLE
Support for regular expressions (regex, grep)

### DISCUSSION THREAD

**Discussion by mprewitt** [massCodeIO/massCode#184] [Ideas]
It would be nice to have support for regular expressions. They are useful in many languages (as well as applications outside of coding, such as the InDesign page layout software). I have snippets that are just the regular expressions themselves.

**Comment by antonreshetov** [comment]
I think in [v3.0](https://github.com/massCodeIO/massCode/discussions/185) it will be easier to solve such issues.

**Reply by antonreshetov** [reply]
Added RegExp support in v3
![image](https://user-images.githubusercontent.com/17480987/183036234-714b1127-a926-405f-b026-f299abcfbf10.png)
